python/pyarrow/array.pxi

```diff
@@ -159,9 +159,10 @@ def array(object obj, type=None, mask=None, size=None, from_pandas=None,
 
     Notes
     -----
-    Localized timestamps will currently be returned as UTC (pandas's native
-    representation). Timezone-naive data will be implicitly interpreted as
-    UTC.
+    Timezone will be preserved in the returned array for timezone aware data,
+    else no timezone data will be returned for naive timestamps.
+    Internally, UTC values are stored for timezone aware data,
```

Comment:
Maybe `timezone aware` --> `timezone-aware` for symmetry with `timezone-naive`.

Reply:
@edponce Made the suggested change.